core/identity-management/iin-agent/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: '2'
+services:
+  iin-agent:
+    image: ${DOCKER_IMAGE_NAME}:${DOCKER_TAG}
+    container_name: iin-agent-${MEMBER_ID}-${SECURITY_DOMAIN}
+    environment:
+      - SECURITY_DOMAIN=${SECURITY_DOMAIN}
+      - MEMBER_ID=${MEMBER_ID}
+      - IIN_AGENT_ENDPOINT=0.0.0.0:${IIN_AGENT_PORT}
+      - WEAVER_CONTRACT_ID=${WEAVER_CONTRACT_ID}
+      - IIN_AGENT_TLS=${IIN_AGENT_TLS}
+      - IIN_AGENT_TLS_CERT_PATH=${IIN_AGENT_TLS_CERT_PATH}
+      - IIN_AGENT_TLS_KEY_PATH=${IIN_AGENT_TLS_KEY_PATH}
+      - DLT_TYPE=${DLT_TYPE}
+      - CONFIG_PATH=/opt/iinagent/config.json
+      - DNS_CONFIG_PATH=/opt/iinagent/dnsconfig.json
+      - SECURITY_DOMAIN_CONFIG_PATH=/opt/iinagent/security-domain-config.json
+      - SYNC_PERIOD=${SYNC_PERIOD}
+      - AUTO_SYNC=${AUTO_SYNC}
+    volumes:
+      - ${DLT_SPECIFIC_DIR}:/opt/iinagent/extra
+      - ${CONFIG_PATH}:/opt/iinagent/config.json
+      - ${DNS_CONFIG_PATH}:/opt/iinagent/dnsconfig.json
+      - ${SECURITY_DOMAIN_CONFIG_PATH}:/opt/iinagent/security-domain-config.json
+      - ../../relay/credentials:/opt/iinagent/credentials
```

Comment:
I know this is a copy from the Fabric driver's compose file, but it was a hack there to begin with. Can we replace the prefix with an environment variable just like in the preceding lines? That way the creation of this container won't be tied to a hardcoded path for the relay's configuration.

And I think we should make the same change in the Fabric driver (and other places where this is hardcoded.)